rules/detekt/src/test/kotlin/com/twitter/compose/rules/detekt/ComposeMultipleContentEmittersCheckTest.kt

```diff
@@ -97,11 +100,11 @@ class ComposeMultipleContentEmittersCheckTest {
                 }
                 @Composable
                 fun Something3() {
-                    Text("Hi")
+                    Potato()
```

Comment:
why are you using Potatos with Bananas? 😹

Reply:
It makes me chuckle when I see them 🥔 🍌